packages/react-components/react-toast/library/src/components/Toast/Toast.types.ts

```diff
@@ -21,4 +22,5 @@ export type ToastProps = ComponentProps<ToastSlots> & {
  */
 export type ToastState = ComponentState<ToastSlots> & {
   backgroundAppearance: BackgroundAppearanceContextValue;
+  intent: ToastIntent | undefined;
```

Comment:
```suggestion
  intent?: ToastIntent | undefined;
```

Otherwise it will be a breaking change

Reply:
Updated